lib/utils.pm

```diff
@@ -154,11 +154,14 @@ sub wait_boot {
         assert_screen 'displaymanager', 200;
         wait_idle;
         if (get_var('DM_NEEDS_USERNAME')) {
-            type_string $username;
+            type_string "$username\n";
         }
```

Comment:
(elsif still makes sense though - how could you miss that when you merged this in first_boot :) )